modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -2343,6 +2363,8 @@ return view.extend({
 				return render_network_status(inst);
 		};
 
+		m.zones = zones;
```

Comment:
Is this assignment needed? It it is, you should solve it in another way; correct me if I'm wrong but it looks like you're manipulating the prototype for [LuCi.form.Map](https://openwrt.github.io/luci/jsapi/LuCI.form.Map.html) and adding a new field which is a big no-no.

Reply:
Good point. I'll rework it